packages/arb-util/configuration/configuration.go

```diff
@@ -321,12 +318,43 @@ type Config struct {
 	MetricsServer Metrics `koanf:"metrics-server"`
 }
 
+// DefaultCoreSettings is useful in unit tests
+func DefaultCoreSettings() *Core {
+	return &Core{
+		Cache: CoreCache{
+			BasicInterval: 100,
+			BasicSize:     1000,
+			LRUSize:       1000,
+			TimedExpire:   20 * time.Minute,
+		},
+		CheckpointGasFrequency:    1_000_000,
+		CheckpointLoadGasCost:     1_000_000,
+		CheckpointLoadGasFactor:   4,
+		CheckpointMaxExecutionGas: 1_000_000_000,
+		MessageProcessCount:       10,
+	}
+}
+
+// DefaultNodeSettings is useful in unit tests
+func DefaultNodeSettings() *Node {
+	return &Node{
+		Cache: NodeCache{
+			AllowSlowLookup: true,
+			LRUSize:         1000,
+			TimedExpire:     20 * time.Minute,
+		},
+		LogProcessCount: 100,
+		LogIdleSleep:    10 * time.Millisecond, // 10 for dev, 100 for server
+	}
+
+}
+
 func (c *Config) GetNodeDatabasePath() string {
 	return path.Join(c.Persistent.Chain, "db")
 }
 
 func (c *Config) GetValidatorDatabasePath() string {
-	return path.Join(c.Persistent.Chain, "validator_db")
+	return path.Join(c.Persistent.Chain, "validator-db")
```

Comment:
Do we even want a separate validator db to begin with? It should just be a subset of the node's db, right? So they should be somewhat interchangeable.